{{cookiecutter.profile_name}}/lsf_submit.py

```diff
@@ -86,10 +86,20 @@ def memory_units(self) -> Unit:
     def resources_cmd(self) -> str:
         mem_in_clusters_units = self.mem_mb.to(self.memory_units)
         mem_value_to_submit = math.ceil(mem_in_clusters_units.value)
-        return (
-            "-M {mem} -n {threads} "
-            "-R 'select[mem>{mem}] rusage[mem={mem}] span[hosts=1]'"
-        ).format(mem=mem_value_to_submit, threads=self.threads)
+        resources_str = (
+            "-M {mem} -n {threads} -R 'select[mem>{mem}] "
+            "rusage[mem={mem}] span[hosts=1]'".format(
+                mem=mem_value_to_submit, threads=self.threads
+            )
+        )
+
+        for time_str in ("time", "runtime", "walltime"):
+            if self.resources.get(time_str, False):
+                time_in_mins = int(self.resources[time_str])
+                resources_str += " -W {:02d}:{:02d}".format(
+                    time_in_mins // 60, time_in_mins % 60
```

Comment:
Nice solution. However, I am wondering if this is necessary? From the [LSF docs](https://www.ibm.com/support/knowledgecenter/en/SSWRJV_10.1.0/lsf_command_ref/bsub.__w.1.html) it says you can just enter this as the number of minutes

> The run limit is in the form of [hour:]minute. The minutes can be specified as a number greater than 59. For example, three and a half hours can either be specified as 3:30, or 210.

Reply:
Right again, it's unnecessary. Would be a bit more in accordance with slurm where it's `HH:MM:SS`, but I simplified it to `MM` only.